nixos/lib/test-driver/default.nix

```diff
@@ -41,4 +42,8 @@ python3Packages.buildPythonApplication {
     echo -e "\x1b[32m## run black\x1b[0m"
     black --check --diff .
   '';
+
+  passthru.tests = {
+    inherit (nixosTests.nixos-test-driver) driver;
```

Comment:
Should we add all of them?

Reply:
All of them := all of the driver.*
All of them := nixosTests.nixos-test.driver.* ?

Either case, more tests is always good in my book and the test driver has suffered from several backlashes in the past due to excessively (good I guess) type checking and formatting.

If you suggest me to add all of them, will do so.